web-performance/README.md

```diff
@@ -0,0 +1,28 @@
+Web Performance
+===============
```

Comment:
I think we tend to use the atx-style Markdown headings (using hashes) these days:

```
# Web Performance
```

The setext-style (underlines) limits us to only two heading levels.